pkg/server/server.go

```diff
@@ -55,12 +55,12 @@ func (s *Config) analyzeHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = config.RunAnalysis()
-	if err != nil {
-		color.Red("Error: %v", err)
+	analysisErrors := config.RunAnalysis()
+	if analysisErrors != nil {
+		for _, err := range analysisErrors {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
Won't the first http Error return the function?

Reply:
Bien vu ! Je vais modifier cela